api/src/MemberMatch/Member.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RaceResults.MemberMatch
+{
+    public class Member
+    {
+        public List<string> FirstList;
+        public List<string> LastList;
+        public string City;
+
+        public static List<string> ProcessName(string field)
```

Comment:
Can you add a docstring to this method? I'm not sure what `field` means in this context and what this method is supposed to return

Reply:
This will be fixed in the next pull request.